STM32Resources/README.adoc

```diff
@@ -0,0 +1,66 @@
+= Compiling and Flashing STM Board on Fedora
+
+. Go to UCSolarCarTeam repo > Recruit Resources, and clone.
```

Comment:
We should add the resources to our own repo in this PR.

Reply:
Yeah, we didn't have the files. That's why I cloned it from there